.prettierignore

```diff
@@ -39,4 +39,7 @@ dist
 
 megalinter-reports/
 
-pnpm-lock.yaml
\ No newline at end of file
+pnpm-lock.yaml
+
+# MDX v3
+*.mdx
```

Comment:
I'm ok with this.
Let's track https://github.com/prettier/prettier/issues/12209 until it's fixed.